content/en/examples/service/networking/custom-dns.yaml

```
@@ -10,7 +10,7 @@ spec:
   dnsPolicy: "None"
   dnsConfig:
     nameservers:
-      - 1.2.3.4
+      - 192.0.2.1
```

To address @tengqm's [reservation](https://github.com/kubernetes/website/pull/39169#issuecomment-1409944922), how about:
```suggestion
      - 192.0.2.1 # this is an example
```

_or_ use an AS112 IP address, such as 10.42.0.1?

```suggestion
      - 10.42.0.1 # this is an example
```

How about either of those approaches? I notice that the IPv6 example is already using a documentation IP address.

---

Adding a comment is good and clear enough.
Thanks.